tests/probe/t15-specimen.sh

```
@@ -0,0 +1,66 @@
+#!/bin/bash
+# Throwaway specimen for the T15 probe (issue #108). Deliberately defective in
+# two well-separated regions so CodeRabbit opens a thread on each. Region A is
+# rewritten in a later commit to outdate its thread; region B is left untouched
+# and stays live. Delete with the branch.
+
+# ---------------------------------------------------------------- region A ---
+
+sum_sizes() {
+  DIR=$1
+  total=0
+  cd $DIR
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)